### ISSUE TITLE
Publish a new release [0.3.0]

### ISSUE THREAD

**Issue by bgrnwdvg** [AWegnerGitHub/stackapi#50]
Hi @AWegnerGitHub, would it be possible to publish a new release of `StackAPI` to PyPI so others can use the changes included in #49 ?

**Comment by AWegnerGitHub** [comment]
```
$ python3 -m pip index versions stackapi
WARNING: pip index is currently an experimental command. It may be removed/changed in a future release without prior warning.
stackapi (0.3.0)
Available versions: 0.3.0, 0.2.0, 0.1.12, 0.1.9, 0.1.8, 0.1.7, 0.1.6
```

`0.3.0` is now available. I've also bumped the default API from `2.2` to `2.3`.

**Comment by bgrnwdvg** [comment]
Thanks!
